src/lib/components/Communities/CommunityHeader.js

```diff
@@ -31,15 +31,22 @@ class CommunityHeaderButton extends React.Component {
 class CommunityHeaderInfo extends React.Component {
   render() {
     const { community, imagePlaceholderLink } = this.props;
+    // TODO: Remove checks below once community is resolved in the backend
+    const communityLogo = community?.links?.logo
+      ? community.links.logo
+      : imagePlaceholderLink;
+    const communityTitle = community?.metadata?.title
+      ? community.metadata.title
+      : community;
     return (
       <Container className="community-header">
         <Image
           className="community-logo-header"
-          src={community.links.logo}
+          src={communityLogo}
```

Comment:
Do you think this could also be

```js
src={community?.links?.logo}
```

because it should use the fallbackSrc if it is undefined right?

Reply:
That's what I thought but doesn't seem the case, if it's undefined it will display a broken img.